### PR TITLE
Add rbac users for search benchmark script

### PR DESCRIPTION
**Pull request by zlayne**
Added rbac users:
- search-limited-access-user: access to only 10 clusters (not including local-cluster)
- search-wide-access-user: access to n - 10 clusters (not including local-cluster)

Updated search query to use python requests lib for better response handling.

Better error handling on search queries for transparency in test logs. *Graphql can return errors as part of a successful request that previously were not being handled. 

Fixed search query syntax (`\n` chars need to be `\\n`)

Removed default limit for (fixed by adding `limit: -1`)

### CODE REVIEW

**Comment by zlayne** [comment]
FYI @jlpadilla

File: acm-deploy-load/benchmark-search.py [hunk 1]
```diff
@@ -39,28 +44,77 @@ def getUserToken(user):
     output = ""
   return output
 
+def getManagedClusterList():
+  managedClusters = []
+  oc_cmd = ["oc", "get", "managedcluster", "-A", "-o", "json"]
```
Comment (jlpadilla) [inline acm-deploy-load/benchmark-search.py:49]: You could just request the names with:
```
oc get namagedClusters -A -o custom-columns=NAME:metadata.name
```

File: acm-deploy-load/benchmark-search.py [hunk 2]
```diff
@@ -71,25 +124,21 @@ def measureQuery(URL, TOKEN, numRequests, queryData, queryName):
   max = 0
   avg = 0
   for x in range(numRequests):
```
Comment (akrzos) [inline acm-deploy-load/benchmark-search.py:126]: In the future you might want to drop a log line once every 10 or 100 requests so users know that the application is still running.

**Review by akrzos — APPROVED** [review]
LGTM, ran on the LTA.

**Pull request merged**